.github/workflows/package.yml

```diff
@@ -16,4 +16,4 @@ jobs:
         run: cd .scripts && sh ./package.sh
         env:
           AUTHOR_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          REVIEWER_TOKEN: ${{ secrets.REVIEWER_TOKEN }}
+          REVIEWER_TOKEN: ${{ secrets.AFRESHY_PAT }}
```

Comment:
What's the significance of these?

Reply:
We can't merge PRs without a review, so we use another token to approve the upgrade PR. This one is an org-level secret for the Afreshy user. The existing token (taken from the mobile .env) didn't have the necessary permissions.